<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TagsOptionsPage</class>
 <widget class="QWidget" name="TagsOptionsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>539</width>
    <height>425</height>
   </rect>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <widget class="QCheckBox" name="write_tags">
     <property name="text">
      <string>Write tags to files</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="preserve_timestamps">
     <property name="text">
      <string>Preserve timestamps of tagged files</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="rename_files">
     <property name="title">
      <string>Before tagging</string>
     </property>
     <layout class="QVBoxLayout">
      <property name="spacing">
       <number>2</number>
      </property>
      <item>
       <widget class="QCheckBox" name="clear_existing_tags">
        <property name="text">
         <string>Clear existing tags</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="remove_id3_from_flac">
        <property name="text">
         <string>Remove ID3 tags from FLAC files</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="remove_ape_from_mp3">
        <property name="text">
         <string>Remove APEv2 tags from MP3 files</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>6</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="preserved_tags_label">
        <property name="text">
         <string>Preserve these tags from being cleared or overwritten with MusicBrainz data:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="preserved_tags"/>
      </item>
      <item>
       <widget class="QLabel" name="preserved_tags_help">
        <property name="text">
         <string>Tags are separated by spaces, and are case-sensitive.</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="rename_files_2">
     <property name="title">
      <string>Tag compatibility</string>
     </property>
     <layout class="QVBoxLayout">
      <property name="spacing">
       <number>2</number>
      </property>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>ID3v2 version</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QRadioButton" name="write_id3v24">
           <property name="text">
            <string>2.4</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="write_id3v23">
           <property name="text">
            <string>2.3</string>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string/>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>ID3v2 text encoding</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QRadioButton" name="enc_utf8">
           <property name="text">
            <string>UTF-8</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="enc_utf16">
           <property name="text">
            <string>UTF-16</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="enc_iso88591">
           <property name="text">
            <string>ISO-8859-1</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string/>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="write_id3v1">
        <property name="text">
         <string>Also include ID3v1 tags in the files</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>274</width>
       <height>41</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>write_tags</tabstop>
  <tabstop>preserve_timestamps</tabstop>
  <tabstop>clear_existing_tags</tabstop>
  <tabstop>remove_id3_from_flac</tabstop>
  <tabstop>remove_ape_from_mp3</tabstop>
  <tabstop>preserved_tags</tabstop>
  <tabstop>write_id3v24</tabstop>
  <tabstop>write_id3v23</tabstop>
  <tabstop>enc_utf8</tabstop>
  <tabstop>enc_utf16</tabstop>
  <tabstop>enc_iso88591</tabstop>
  <tabstop>write_id3v1</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
